tools/pd-ctl/pdctl/command/component_command.go

```diff
@@ -112,15 +112,27 @@ func deleteComponentConfigCommandFunc(cmd *cobra.Command, args []string) {
 }
 
 func getComponentIDCommandFunc(cmd *cobra.Command, args []string) {
-	if len(args) != 1 {
+	argsLen := len(args)
+	// args too long
+	if argsLen > 1 {
 		cmd.Usage()
 		return
 	}
+	var (
+		ids = "all"
+	)
+	if argsLen == 1 {
+		ids = args[0]
+	}
+	prefix := path.Join(componentConfigPrefix, "ids", ids)
 
-	prefix := path.Join(componentConfigPrefix, "ids", args[0])
 	r, err := doRequest(cmd, prefix, http.MethodGet)
 	if err != nil {
-		cmd.Printf("Failed to get component %s's id: %s\n", args[0], err)
+		if argsLen > 0 {
+			cmd.Printf("Failed to get component %s's id: %s\n", args[0], err)
```

Comment:
Oh, you can change to `cmd.Println`.

Reply:
`cmd.Println` cannot be used because 2 parameters need to be formatted.

Reply:
Yes. Line 132 can be changed.

Reply:
modified.